controllers/EntryController.php

```diff
@@ -303,6 +304,23 @@ public function actionParticipantsList($id = null)
         ]);
     }
 
+    /**
+     * Action to export participants
+     *
+     * @param int $id
+     * @param int|null $state
+     * @param string $type File format type: 'csv' or 'xlsx'
+     * @return Response
+     */
+    public function actionExportParticipants(int $id, ?int $state, string $type)
+    {
+        if (!in_array($type, ['csv', 'xlsx'])) {
```

Comment:
Restrict to canEdit only?

Reply:
@luke- This action is rescricted only by `CalendarEntry::find()->readable()`, same as the action "Participants List".
Do you think this new action "Export Participants" should be available only for user who can edit a calendar entry?

Reply:
@yurabakhtin Yes, the Modal with the Button is also only available to Editor or?

Reply:
@luke- No, please see what user can see only with view permissions:

![cal-no-edit-perm](https://github.com/humhub/calendar/assets/6995524/dee57144-e3bc-4259-bbb4-aa41703dbad1)

and user with edit perms:

![cal-has-edit-perm](https://github.com/humhub/calendar/assets/6995524/5af6c410-a172-48d6-bfb3-f5290e7d0ab0)

Reply:
@yurabakhtin Thanks, for clarify. 

@Semir1212 Should we hide this button for regular participants? Yes?

Reply:
@luke- @yurabakhtin If users do not have the permission to export the data the button should not be visible to them.

Reply:
@luke- Ok, done in commit https://github.com/humhub/calendar/pull/422/commits/d151d85f7e14a9db8a19b4855c0a83293ac97c35.